paywall/next.config.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs')
+const { join } = require('path')
+const { promisify } = require('util')
+const withSourceMaps = require('@zeit/next-source-maps')
+
+const copyFile = promisify(fs.copyFile)
+
+module.exports = withSourceMaps({
+  publicRuntimeConfig: {
+    unlockEnv: process.env.UNLOCK_ENV || 'dev',
+    httpProvider: process.env.HTTP_PROVIDER || '127.0.0.1',
+    readOnlyProvider: process.env.READ_ONLY_PROVIDER,
+    locksmithHost: process.env.LOCKSMITH_HOST,
+  },
+  webpack: config => {
+    // Fixes npm packages that depend on `fs` module
+    config.node = {
+      fs: 'empty',
+    }
+
+    return config
+  },
+  exportPathMap: async (defaultPathMap, { dev, dir, outDir }) => {
+    // Export robots.txt and humans.txt in non-dev environments
+    if (!dev && outDir) {
+      await copyFile(
+        join(dir, '..', 'unlock-app', 'src', 'static', 'robots.txt'),
+        join(outDir, 'robots.txt')
```

Comment:
I think this can be pruned

Reply:
I think we will want robots.txt, nobody should crawl the paywall